frontend/packages/dev-console/integration-tests/support/pageObjects/add-flow-po.ts

```diff
@@ -105,18 +105,18 @@ export const catalogPO = {
   cardType: 'span.pf-c-badge',
   create: 'button[type="submit"]',
   cancel: '[data-test-id="reset-button"]',
-  cardList: '[role="rowgroup"]',
+  cardList: '[role="grid"]',
   cardHeader: '.pf-c-badge.pf-m-read',
   groupByMenu: 'pf-c-dropdown__menu',
   catalogTypeLink: 'li.vertical-tabs-pf-tab.shown.text-capitalize.co-catalog-tab__empty',
   catalogTypes: {
     operatorBacked: '[data-test="kind-cluster-service-version"]',
-    helmCharts: 'a[href="/?catalogType=HelmChart"]',
-    builderImage: 'ul:nth-child(3) > li:nth-child(1) > a', // This needs to be changed
-    template: 'a[href="/?catalogType=Template"]',
+    helmCharts: 'a[href$="catalogType=HelmChart"]',
+    builderImage: 'a[href$="catalogType=BuilderImage"]',
+    template: 'a[href$="catalogType=Template"]',
```

Comment:
If you like you can also add the following data-tests attributes and change this here to this new data-test references. Wdyt?

```suggestion
    helmCharts: '[data-test="tab HelmChart"]',
    builderImage: '[data-test="tab BuilderImage"]',
    template: '[data-test="tab Template"]',
```